scripts/build.sh

```diff
@@ -13,82 +15,99 @@ check_command go
 check_command 7z
 check_command upx
 
+set -x
+
+export CGO_ENABLED=0
+
 # Windows amd64
 export GOOS=windows
 export GOARCH=amd64
 FILENAME="go2rtc_win64.zip"
-go build -ldflags "-s -w" -trimpath && 7z a -mx9 -bso0 -sdel $FILENAME go2rtc.exe
+go build -ldflags "-s -w" -trimpath
```

Comment:
I did not change anything in these commands, I just broke the `&&` into a new line.